simplefs.h

```diff
@@ -27,6 +27,7 @@
 #include <linux/version.h>
 
 #define USER_NS_REQUIRED() LINUX_VERSION_CODE >= KERNEL_VERSION(5,12,0)
+#define USER_NS_REQUIRED_6_3() LINUX_VERSION_CODE >= KERNEL_VERSION(6,3,0)
```

Comment:
`USER_NS_REQUIRED_6_3` is not an appropriate name. Instead, check the kernel commit message and summarize the reason why kernel developers changed the usage. Finally, reflect the considerations here.